index.js

```diff
@@ -9,8 +10,10 @@ module.exports = {
       'jquery-sticky': function() {
         // The jQuery plugin causes FastBoot to crash, so exclude there
         return {
-          enabled: !process.env.EMBER_CLI_FASTBOOT,
-          import: ['jquery.sticky.js']
+          import: ['jquery.sticky.js'],
+          processTree(input) {
+            return fastbootTransform(input);
+          }
```

Comment:
This could be simplified a little now that it's not relying on `process.env` to something like:

```js
'jquery-sticky': {
  import: {
    include: ['jquery.sticky.js'],
    processTree(input) {
      return fastbootTransform(input);
    }
  }
}
```